src/libraries/System.Private.CoreLib/src/System/IO/FileSystem.Unix.cs

```diff
@@ -51,6 +89,18 @@ public static void CopyFile(string sourceFullPath, string destFullPath, bool ove
             }
         }
 
+        private static void StandardCopyFile(SafeFileHandle src, SafeFileHandle dst, long fileLength)
```

Comment:
This doesn't need a helper.  The call sites can just use e.g.:
```C#
Interop.CheckIo(Interop.Sys.CopyFile(src, dst!, fileLength));
```
instead of:
```C#
StandardCopyFile(src, dst!, fileLength);
```
It's not saving enough to be worth a separate method that somewhat obscures what it's actually doing.

Reply:
I had this in a helper to ensure that if unix changes for example, that macOS doesn't get left behind, since it would only be noticed when copying between different volumes most likely. I will instead add a comment above both instances saying that they need to be kept in sync.